day_7/10_little_monkeys.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
@MichaelEWalker87 - This solution to 10_little_monkeys is a little difficult to parse.  Also, when I ran the function in the console, the first verse did not print out ("Ten little monkeys...") - it started with "Nine little monkeys...".  Additionally the `addMonkey` and `minusMonkey` methods are mutating the `monkeyNumber` array (changing its contents) which we normally try to avoid in JS because it makes it hard to find errors/bugs down the line.

Could you brainstorm and update with an alternate way of solving this?  For example, what about trying the following?  Does it log correctly on each iteration?

```
var monkeys = [
  'Ten',
  'Nine',
  'Eight',
  'Seven',
  'Six',
  'Five',
  'Four',
  'Three',
  'Two',
  'One'
];

for (var i = 0; i < monkeys.length; i++) {
  if (monkeys[i] === 'One') {
    console.log("One little monkey jumping on the bed, \n He fell off and bumped his head, \n Mama called the doctor and the doctor said,\n\"Get those monkeys right to bed!\"");
  } else {
    console.log(monkeys[i] + " little monkeys jumping on the bed, \n One fell off and bumped his head, \n Mama called the doctor and the doctor said,\n \"No more monkeys jumping on the bed!\"");
  }
}
```